### PR TITLE
Print method

### PR DESCRIPTION
**Pull request by aghaynes**
Add a print method for the objects to prevent simply all the data being shown

### CODE REVIEW

File: R/load_secuTrial_export.R [hunk 1]
```diff
@@ -117,3 +118,46 @@ load_secuTrial_export <- function(data_dir, tables = "all", add_id_name = NULL)
...
+  tab$original_name <- rownames(tab)
+  rownames(tab) <- NULL
+  print(tab, row.names = FALSE)
+
+}
+
+#' @rdname secuTrialdata
+#' @usage plot(x)
+plot.secuTrialdata <- function(x){
+  vs <- visitstructure(x)
```
Comment (PatrickRWright) [inline R/load_secuTrial_export.R]: should this be visit_structure?

Comment (aghaynes) [reply R/load_secuTrial_export.R]: No, it's the plot method for the secuTrialdata class. There is also a plot method for secuTrialvisit objects.

Comment (aghaynes) [reply R/load_secuTrial_export.R]: ah...you mean line 161.... yes i think it probably should... well spotted

**Comment by PatrickRWright** [comment]
@aghaynes 
1. Can you have a quick look at the README? I was not quite sure about the merge conflict.
2. I have tracked #11 issue since I can't test visit_structure(). I guess we can merge it for now and maybe I can look at providing some test data.

**Comment by aghaynes** [comment]
done, but it looks like the MD needs knitting again... looks to me as if the package is outdated (factorize wasn't found)

**Pull request merged**